js/msp.js

```diff
@@ -1510,6 +1506,9 @@ MSP.crunch = function (code) {
             for (var i = 0; i<CONFIG.name.length; i++) {
```

Comment:
FC will only handle the maximum allowed number of characters ([See serial_msp.c line 1882 here](https://github.com/betaflight/betaflight/pull/747/files#diff-b2b29e72f4b7263e160784dff0c923d7R1882)) and the rest is truncated. 
If we add a limit/check also in the configurator then it will creata a dependency to update configurator/FW at the same time, which I think we should avoid ?